tests/agent-opentracing/src/test/java/org/hawkular/apm/tests/agent/opentracing/jms/JavaxJMSITest.java

```diff
@@ -87,28 +85,15 @@ public void testQueueRequestOnly() throws JMSException, InterruptedException, Ex
 
         assertEquals(JMSAsyncServer.TEST_MESSAGE, serverResult.get());
 
-        Wait.until(() -> getApmMockServer().getTraces().size() == 2);
-
-        assertEquals(2, getApmMockServer().getTraces().size());
-
-        Trace client = getApmMockServer().getTraces().stream().filter(t -> t.initialFragment()).findFirst()
-                .orElse(null);
-        Trace server = getApmMockServer().getTraces().stream().filter(t -> !t.initialFragment()).findFirst()
-                .orElse(null);
-
-        assertEquals(client.getTraceId(), server.getTraceId());
-        assertEquals(1, client.getNodes().size());
```

Comment:
This was because of the Trace/Node structure - so only one node inside the Trace fragment. The equivalent is just a single span.